resources/dicts/lead_ceremony_df.json

```diff
@@ -447,22 +447,16 @@
       "rank": ["warrior"],
       "life_giving": [
         {
-          "text": "EXAMPLE r_c walks up to m_c, offering a life for [virtue]. They smile, and state that the Clan will do well under m_c's leadership.",
-          "virtue": ["ambition", "bravery", "certainty", "clear judgement", "confidence", "courage",
-      "determination", "endurance", "farsightedness", "instincts", "strength", "ferocity", "cunning", "shamelessness", "zealotry", "bloodlust", "restless energy", 
-      "stubborness", "authority"]
+          "text": "r_c meets m_c's eyes with a smirk, sliding through the murk like a claw through freshkill to give a life of [virtue]. {PRONOUN/r_c/subject/CAP} {VERB/r_c/hiss/hisses} quiet praise to m_c's more reckless tendencies, encouraging the new leader to think with {PRONOUN/m_c/poss} claws, not {PRONOUN/m_c/poss} head.",
```

Comment:
can you change the "life of [virtue]" to "life for [virtue]"?  (also i *love* the "like a claw through freshkill" bit, very descriptive, very cool)